src/NServiceBus.Metrics.PerformanceCounters/Scripts/CreateNSBPerfCounters.ps1

```diff
@@ -1,7 +1,7 @@
 
 #requires -RunAsAdministrator
 Function InstallNSBPerfCounters {
-    
+
```

Comment:
Is `CreateNSBPerfCounters.ps1` the name of the file that the old generation code would create?

Reply:
Yep, now that we're bumping the major we could change it though.

Reply:
I'll leave that up to you.